awx/ui_next/src/screens/Team/TeamAccess/TeamAccessList.jsx

```diff
@@ -80,11 +104,26 @@ function TeamAccessList({ i18n }) {
     return `/${resource_type}s/${resource_id}/details`;
   };
 
+  const isSysAdmin = roles.some(role => role.name === 'System Administrator');
+  if (isSysAdmin) {
+    return (
+      <EmptyState variant="full">
+        <EmptyStateIcon icon={CubesIcon} />
+        <Title headingLevel="h5" size="lg">
+          {i18n._(t`System Administrator`)}
+        </Title>
+        <EmptyStateBody>
+          {i18n._(t`System administrators have access to all permissions.`)}
```

Comment:
Can we rephrase this? It's understandable, but I want to be more specific. How about "System administrators have full permissions and access".